openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -314,7 +321,7 @@ def get(self, *args, **kwargs):  # pylint: disable=unused-argument
         user_qs = get_org_users_qs(self.request.user)
         user_ids = user_qs.values_list('id', flat=True)
 
-        completed_count, in_progress_count = get_completed_course_count_filters(user=self.request.user)
+        completed_count, in_progress_count = get_completed_course_count_filters(True, user=self.request.user)
```

Comment:
should add variable name as well for better readability?
`get_completed_course_count_filters(exclude_staff_superuser=True, user=self.request.user)`